### PR TITLE
image-layout.md: clarify the layout file name MUST be `oci-layout`

### PR DESCRIPTION
**Pull request by coolljt0725**
Signed-off-by: Lei Jitang <leijitang@huawei.com>

### CODE REVIEW

File: image-layout.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The image layout MUST contain two top level directories:
 
 Both `blobs` and `refs` MAY be empty.
 
-The image layout MUST also contain a file that is used to identify the layout version:
+The image layout MUST also contain a file that is used to identify the layout version, the file name MUST be `oci-layout`:
 
 - `oci-layout` MUST contain a JSON object with a version field `{"imageLayoutVersion": "1.0.0"}` and MAY include additional fields.
```
Comment (wking) [inline image-layout.md]: How about rephrasing both of these lines to something like:

> The image layout MUST also contain an `oci-layout` file that is used to identify the layout version.
> `oci-layout` MUST contain a JSON object with `imageLayoutVersion` set to `1.0.0`.
> `oci-layout` MAY include additional fields.

Comment (coolljt0725) [inline image-layout.md]: I'm ok with this rephrasing.

**Comment by coolljt0725** [comment]
updated per @wking 's suggestion

**Comment by coolljt0725** [comment]
This is a dup of #434  closing

**Pull request closed without merging**